react-firebase-email-verification.md

```diff
@@ -0,0 +1,412 @@
++++
+draft = true
+title = "Email Verification with Firebase in React"
+description = "A Firebase React tutorial on how to enable email verification. Only users that confirmed their email address with a email confirmation have access to your application. Every other user who is using a fake email is not authorized ..."
+date = "2018-11-30T07:52:46+02:00"
+tags = ["React", "JavaScript"]
+categories = ["React", "JavaScript"]
+keywords = ["react firebase email verification", "react firebase email confirmation", "react firebase double opt in"]
+news_keywords = ["react firebase email verification", "react firebase email confirmation", "react firebase double opt in"]
+hashtag = "#ReactJs"
+card = "img/posts/react-firebase-email-verification/banner_640.jpg"
+banner = "img/posts/react-firebase-email-verification/banner.jpg"
+contribute = "react-firebase-email-verification.md"
+headline = "Email Verification with Firebase in React"
+
+summary = "A Firebase React tutorial on how to enable email verification. Only users that confirmed their email address with a email confirmation have access to your application. Every other user who is using a fake email is not authorized."
++++
+
+{{% sponsorship %}}
+
+{{% pin_it_image "react firebase email verification" "img/posts/react-firebase-email-verification/banner.jpg" "is-src-set" %}}
+
+{{% react-firebase-book %}}
+
+{{% read_before_5 "This tutorial is part 6 of 6 in this series." "Part 1:" "A Firebase in React Tutorial for Beginners" "https://www.robinwieruch.de/complete-firebase-authentication-react-tutorial" "Part 2:" "React Firebase Authorization with Roles" "https://www.robinwieruch.de/react-firebase-authorization-roles-permissions" "Part 3:" "React Firebase Auth Persistence with Local Storage" "https://www.robinwieruch.de/react-firebase-auth-persistence" "Part 4:" "React Firebase Social Login: Google, Facebook, Twitter" "https://www.robinwieruch.de/react-firebase-social-login" "Part 5:" "React Firebase: Link Social Logins" "https://www.robinwieruch.de/react-firebase-link-social-logins" %}}
+
+In your application, users can employ an email/password combination, but also social logins to get access to your service or product. Often, the email address associated with the social logins is confirmed by the social platform (Google, Facebook, Twitter) and you know this email address really exists. But what about the email address used with the password? Because users are sometimes unwilling to provide real email addresses, they'll simply make one  up, so you can't provide them with further information via email or to integrate them with third-parties where a valid email address is required. In this section, I will show you how to confirm user email addresses before they can access your application. After an email verification with a double opt-in send by email, users are authorized to use your application.
```

Comment:
"one  up" <- double spaces

Reply:
I don't see the phrase "one up" in the text, not could I find it with a command+F search.

Reply:
"they'll simply make one  up, so" <- I can fix it myself 👍